### PR TITLE
Add ability to link to page with search filled in #7

### PR DESCRIPTION
**Pull request by KenyanJesus**
- Added favicon
- Added labs from https://docs.google.com/spreadsheets/d/1n3I0gGrNMDW-q6iwUcQszhITXwToWHV64EWHjaOQNLI/edit#gid=0
- Added hashing to URLs allowing for search linking

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -24,7 +25,6 @@ $(document).ready( function () {
 			{ data: 'city' },
 			{ data: 'country' },
 			{ data: 'process35',
-			  render: emoj,
```
Comment (sams96) [inline script.js]: Why was this removed?

Comment (KenyanJesus) [reply script.js]: Mistake
